CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.0.1] - 2021-02-22
+### Added
+- First release of Lab Results Adaptor
+- NHSACK is not produced
```

Comment:
Not sure about "Added: this thing is not added"
Maybe list that NHSACK is not produced as a known issue?

Reply:
It has already been released. We'll remove that with v0.2